src/interfaces/records-write.ts

```diff
@@ -111,21 +112,21 @@ export type CreateFromOptions = {
   published?: boolean;
   messageTimestamp?: string;
   datePublished?: string;
-  authorizationSignatureInput?: SignatureInput;
-  attestationSignatureInputs?: SignatureInput[];
+  authorizationSigner?: Signer;
+  attestationSigners?: Signer[];
   encryptionInput?: EncryptionInput;
 };
 
 export class RecordsWrite {
   private _message: InternalRecordsWriteMessage;
   /**
    * Valid JSON message representing this RecordsWrite.
-   * @throws `DwnErrorCode.RecordsWriteMissingAuthorizationSignatureInput` if the message is not signed yet.
+   * @throws `DwnErrorCode.RecordsWriteMissingauthorizationSigner` if the message is not signed yet.
    */
   public get message(): RecordsWriteMessage {
     if (this._message.authorization === undefined) {
       throw new DwnError(
-        DwnErrorCode.RecordsWriteMissingAuthorizationSignatureInput,
+        DwnErrorCode.RecordsWriteMissingauthorizationSigner,
```

Comment:
super nit: capitalization
```suggestion
        DwnErrorCode.RecordsWriteMissingAuthorizationSigner,
```

Reply:
Not nit at all, keen eyes! Fixing!